legal-api/src/legal_api/services/authz.py

```diff
@@ -358,7 +391,7 @@ def get_allowed_filings(business: Business, state: Business.State, legal_type: s
         state_filing = Filing.find_by_id(business.state_filing_id)
 
     # doing this check up front to cache result
-    has_blocker_default = business_blocker_default_check(business)
+    has_blocker_default: dict = business_blocker_default_check(business)
```

Comment:
please rename `has_blocker_default` to `has_business_blocker_dict`

Reply:
okay